test/parallel/test-module-isBuiltIn.js

```diff
@@ -7,7 +7,7 @@ const { isBuiltIn } = require('module');
 assert(isBuiltIn('http'));
 assert(isBuiltIn('sys'));
 assert(isBuiltIn('node:fs'));
-assert(isBuiltIn('node:test'));
+assert(!isBuiltIn('node:test'));
```

Comment:
node:test is a builtin; this doesn't seem right

Reply:
[loader.js#L183](https://github.com/nodejs/node/blob/main/lib/internal/modules/cjs/loader.js#L183-L189) is where the builtins are created and `node:test` is in [schemelessBlockList](https://github.com/nodejs/node/blob/a36a5469c27a60bc2d9ff590fbf145d59529d9d6/lib/internal/bootstrap/loaders.js#L124)!

Reply:
Right - test isn’t a builtin, but node:test is